.circleci/config.yml

```diff
@@ -596,3 +646,14 @@ workflows:
               only: main
             tags:
               only: /^v.*/
+      # DockerHub publication for open pull requests
+      #
+      # Publish docker images only if all build, lint and test jobs succeed
+      - pull-request-hub:
+          requires:
+            - build-docker
+            - test-back
+            - tray
+          filters:
```

Comment:
isn't it going to build on all pushes even when there is no pull request?

Reply:
in this project, the CI only builds pull requests so the CI will not not run if there is not a PR open.

Reply:
:+1: